chaosinstana/api.py

```diff
@@ -43,6 +43,6 @@ def get_all_events(from_time: str, to_time: str,
                 secrets.get("instana_api_token"))
         }
     )
-    logger.debug("r.json(): {}".format(r.json()))
+    # logger.debug("r.json(): {}".format(r.json()))
```

Comment:
If this is not needed, it should be removed.

Reply:
Is it useful to have the instana response logged to debug?

Reply:
I can see the need for logging whether the call was successful as info level, maybe response if debug level as you have it?